modules/get-started/pages/quick-start.adoc

```diff
@@ -36,111 +55,343 @@ docker compose version
+
+You can filter messages in topics using several methods:
+
+- Text search: Search for specific text within the message.
+- Partition offset filters: Set the start and end offsets to narrow down the message range.
+- JavaScript filters: Use JavaScript function bodies to create complex filters that run server-side.
+
+Suppose you're asked to find all transactions related to the `.edu` domain. You can use a JavaScript filter to display only messages that include email addresses in that domain.
+
+. In the left menu, click *Topics*.
+// (step {"action":"find", "selector": "[data-testid='nav-link-Topics']", "matchText": "Topics", "click": true})
+. Click the *transactions* topic. When the topic's page opens, the *Messages* tab is selected by default.
+// (step {"action":"goTo", "url": "http://localhost:8080/topics/transactions"})
+// (step {"action":"wait"})
+. Click *Add filter* > *JavaScript Filter*.
+// (step {"action":"saveScreenshot", "path": "topic.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+// (step {"action":"find", "selector": "[data-testid='add-topic-filter']", "matchText": "Add filter", "click": true, "timeout": 10000})
+// (step {"action":"find", "selector": "[data-testid='add-topic-filter-javascript']", "matchText": "JavaScript Filter", "click": true, "timeout": 10000})
+. Give your filter a name, such as "Find .edu domains".
+// (step {"action":"find", "selector": "[data-testid='add-javascript-filter-name']", "typeKeys": "Find .edu domains", "click": true, "timeout": 10000})
+// (step {"action":"saveScreenshot", "path": "js-filter.png", "directory": "../../modules/console/images", "overwrite": "byVariation", "maxVariation": 10})
++
+image::console:js-filter.png[]
+
+. Replace the default JavaScript code with the following:
++
+[,js]
 ----
-[+] Running 4/4
-⠿ Network redpanda_network                 Created  0.0s
-⠿ Volume "redpanda-quickstart_redpanda-0"  Created  0.0s
-⠿ Container redpanda-0                     Started  0.3s
-⠿ Container redpanda-console               Started  0.6s
+return value.email.includes(".edu");
 ----
---
-Three Brokers::
+
+. Click *Save* to apply the filter.
+// (step {"action":"find", "selector": "[data-testid='add-javascript-filter-save']", "matchText": "Save", "click": true})
 +
---
+You should see only messages that include the specific domain.
+
+By default, the filter runs on the newest 50 messages. You can run the filter on older messages by changing the start offset. For example, to start from the oldest messages, select *Beginning*.
+// (step {"action":"goTo", "url": "http://localhost:8080/topics/transactions"})
+// (step {"action":"find", "selector": ".chakra-tabs__tablist button[data-index='5']", "matchText": "Documentation", "click": true})
+// (step {"action":"wait"})
+// (step {"action":"saveScreenshot", "path": "topic-documentation.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+
+=== Schema Registry
```

Comment:
For consistent style, you could rename this heading **Configure Schema Registry** or **Explore Schema Registry**. Same thing with **Data transforms** heading.